examples/go/echo/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.23.0
 
 replace github.com/TBD54566975/ftl => ../../..
 
-require github.com/TBD54566975/ftl v0.248.0
+require github.com/TBD54566975/ftl v0.0.0-00010101000000-000000000000
```

Comment:
@alecthomas I wasn't 100% sure on this, but noticed that `echo` and `time` had different versions here so I made all of them the same. 😬

Reply:
If a correct `replace` directive exists, the version is irrelevant, so it doesn't hurt.